### PR TITLE
Create msd.yaml

### PR DESCRIPTION
**Pull request by chueatwork**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 38a95c85cb46e9c089d4f87fa105419aa30595d5
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=6582548e-dc5e-4a71-850d-a5ed3fb8a38e%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: datasets/msd [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  - ct
+  - health
+  - life sciences
+  - medicine
+License: "[CC-BY-SA 4.0 International](https://creativecommons.org/licenses/by/4.0/)"
+Resources:
+  - Description: "| Ten tasks from the Medical Segmentation Decathlon Challenge. Tasks are organized by organ system and pathology, as follow: Liver Tumours; Brain Tumours; Hippocampus; Lung Tumours; Prostate; Cardiac; Pancreas Tumour; Colon Cancer; Hepatic Vasculature; Spleen.
+                  Tasks are provided in both tar.gz and uncompressed format."
+    ARN: arn:aws:s3:::MSD-for-MONAI
+    Region: us-east-2
```
Comment (mjorgecardoso) [inline datasets/msd:20]: move to eu-west-1 might be best. Can it be multi-region?

File: datasets/msd [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  - mri
+  - ct
+  - health
+  - life sciences
+  - medicine
+License: "[CC-BY-SA 4.0 International](https://creativecommons.org/licenses/by/4.0/)"
+Resources:
+  - Description: "| Ten tasks from the Medical Segmentation Decathlon Challenge. Tasks are organized by organ system and pathology, as follow: Liver Tumours; Brain Tumours; Hippocampus; Lung Tumours; Prostate; Cardiac; Pancreas Tumour; Colon Cancer; Hepatic Vasculature; Spleen.
+                  Tasks are provided in both tar.gz and uncompressed format."
+    ARN: arn:aws:s3:::MSD-for-MONAI
```
Comment (mjorgecardoso) [inline datasets/msd:19]: let's change the bucket name to "medicalsegmentationdecathlon"

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: bd1e8283256fd8cb47d85df9a0cdcdc0359460af
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=f2674405-3191-4675-a95d-31edd34530d0%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 1da6df6474c9314bdc93ea4bf6d1537628032a41
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=3eeae456-0e10-4d73-b12f-aff0ab4b1b02%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 9ea80df31546890d2abbef2d674c932f14ccd7fd
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=879b9fd4-da6c-43be-9cfb-0166e13bbb24%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: b4de65f176738351d68ffb2d3e3d57b501b7a3e8
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=5b15f7d3-31f3-4304-a992-eccf414d4cc1%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 28902dbccf92dbe3f5bf31ce2d426ae93ff18d39
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=7b1f979c-4c61-4f7a-b1ff-dcea804a029b%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Pull request merged**